src/components/Form/CreateForm.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Form from 'components/Form';
+import CreateFormFields, {
+  formFieldPropTypes,
+} from 'components/Form/CreateFormFields';
+
+function CreateForm({
```

Comment:
```suggestion
function FormBuilder({
```

So that Form is still a prefix?